addons/sale/report/sale_report.py

```diff
@@ -58,6 +68,11 @@ def _get_done_states(self):
 
     order_id = fields.Many2one('sale.order', 'Order #', readonly=True)
 
+    avg_days_to_confirm = fields.Float(
+        'Average Days To Confirm', readonly=True, store=False,  # needs store=False to prevent showing up as a 'measure' option
```

Comment:
```suggestion
        string="Average Days To Confirm", readonly=True, store=False,  # needs store=False to prevent showing up as a 'measure' option
```